apps/ensnode/src/handlers/NameWrapper.ts

```diff
@@ -40,50 +40,51 @@ async function materializeDomainExpiryDate(context: Context, node: Node) {
   }));
 }
 
-async function handleTransfer(
-  context: Context,
-  event: EventWithArgs,
-  eventId: string,
-  tokenId: bigint,
-  to: Address,
-) {
-  await upsertAccount(context, to);
-  const node = tokenIdToNode(tokenId);
+export const makeNameWrapperHandlers = (ownedName: OwnedName) => {
```

Comment:
I had to wrap multiple exports into a factory function that would allow passing extra context to those previously-exported functions. Haven't changed logic of any of these functions.